rdkit/Chem/UnitTestEnumerateHeterocycles.py

```diff
@@ -1,29 +1,20 @@
-
-import doctest
-import unittest
-
-import os
 import csv
+import os
+import unittest
 from random import Random
+
 from rdkit import Chem
 from rdkit.Chem import AllChem
-from rdkit.Chem.EnumerateHeterocycles import (
-    GetHeterocycleReactionSmarts,
-    GetHeterocycleReactions,
-    EnumerateHeterocycles,
-)
+from rdkit.Chem.EnumerateHeterocycles import (EnumerateHeterocycles,
+                                              GetHeterocycleReactions,
+                                              GetHeterocycleReactionSmarts)
+
 
 def has_radical(mol):
-    for atom in mol.GetAtoms():
-        if atom.GetNumRadicalElectrons():
-            return True
-    return False
+    return any(atom.GetNumRadicalElectrons() > 0 for atom in mol.GetAtoms())
```

Comment:
You probably don't need the `> 0`:
```suggestion
    return any(atom.GetNumRadicalElectrons() for atom in mol.GetAtoms())
```